components/net/connector.rs

```diff
@@ -81,7 +81,7 @@ impl Stream for WrappedBody {
                         Decoder::Plain => Some(chunk),
                         Decoder::Gzip(Some(ref mut decoder)) => {
                             let mut buf = vec![0; BUF_SIZE];
-                            *decoder.get_mut() = Cursor::new(chunk.into_bytes());
+                            decoder.get_mut().get_mut().extend(&chunk.into_bytes());
```

Comment:
`&chunk.into_bytes()` should be `chunk.as_ref()`.

Reply:
Done